clients/privacy-center/README.md

```diff
@@ -0,0 +1,53 @@
+# FidesOps Privacy Center
+
+Privacy center for FidesOps. A web application built in Next.js with the FidesUI
+component library.
+
+## Configuration
+
+You may configure the appearance of this web application at build time by modifying the `config.json` file inside the `config` directory. You're able to control:
+
+- The header of the document
+- The descriptive text of the document
+- Which actions are present
+- The titles and descriptions of, and personally identifying information required by, each action
+
+You can also add any CSS you'd like to the page by adding it to the `config.css`
+file inside the `config` directory.
+
+Additionally, if you're handy with React and are feeling a little brave you can
+customize the entire application by modifying the TypeScript source code in
+`pages/index.tsx` and `components/RequestModal.tsx`.
+
+## Development
+
+To serve this application locally, run:
+
+```bash
+npm run dev
+```
+
+## Building
+
+To build this application, run:
+
+```bash
+npm run build
+```
+
+As a Next application, it will output build artifacts to the `.next` directory.
+
+## Testing
+
+To run the interactive test interface, run:
+
+```bash
+npm run test
```

Comment:
Interesting that this is an interactive interface — what's the command here to automatically run all tests? We'll want to add this to the CI jobs too.

Reply:
Great point. I'll add a command for it.

Reply:
(It's simply `jest`, instead of `jest --watch`)